dcat/index.html

```diff
@@ -3863,14 +3863,6 @@ <h4>Property: checksum value</h4>
 
 <h2>Use of inverse properties</h2>
 
-<aside class="ednote">
-<p>This section is a draft for discussion about support for inverse properties in DCAT.</p>
-<p>Related issues:</p>
-
-<div class="issue" data-number="1336"></div>
-
-</aside>
-
 <p>The properties described in <a href="#vocabulary-specification"></a> do not intentionally include inverses, with the purpose of ensuring interoperability also in systems not making use of OWL reasoning.</p>
```

Comment:
Changing as in Alejandra's suggestion
```suggestion
<p>The properties described in <a href="#vocabulary-specification"></a> do not include inverses intentionally, with the purpose of ensuring interoperability also in systems not making use of OWL reasoning.</p>
```